sdk/identity/identity/package.json

```diff
@@ -72,7 +72,7 @@
   "sideEffects": false,
   "dependencies": {
     "@azure/core-http": "^1.0.0",
-    "@azure/core-tracing": "1.0.0-preview.7",
+    "@azure/core-tracing": "1.0.0-preview.8",
```

Comment:
Among all the packages that got touched here, Identity is the only one whose package version was not bumped and so is the same as the published version.

Can you push a commit to update the package version of identity to 1.0.3 and add a changelog entry?

Reply:
This can be done in a separate PR

Reply:
Actually most of affected packages still have the same versions as latest published.

Reply:
Let's have them bumped today